XamlControlsGallery/ControlPages/AnimatedVisualPlayerPage.xaml

```diff
@@ -0,0 +1,107 @@
+﻿<Page
+    x:Class="AppUIBasics.ControlPages.AnimatedVisualPlayerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:animatedvisuals="using:AnimatedVisuals"
+    xmlns:local="using:AppUIBasics"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxcontrols="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+        <local:ControlExample x:Name="AnimatedVisualPlayerExample" 
+                              HeaderText="Playback of a Lottie animation."
+                              >
+
+            <local:ControlExample.Example>
+                <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
+                    <!-- Text -->
+                    <TextBlock Width="600" TextWrapping="WrapWholeWords">
+                        This AnimatedVisualPlayer consumes a Lottie animation which was created in 
+                        Adobe AfterEffects and translated into Windows.UI.Composition objects using 
+                        <Hyperlink NavigateUri="https://aka.ms/lottie">
+                           Lottie-Windows
+                        </Hyperlink>.
+                        Since the 
+                        <Hyperlink NavigateUri="https://docs.microsoft.com/uwp/api/windows.ui.composition.compositionshape">
+                            CompositionShapes
+                        </Hyperlink> being used in the animation are supported on Windows 10 version 17763 and above, 
```

Comment:
Do we need to do anything specific to make this work down-level to the creator's update (RS2)? Or is this handled by the control? In other words, I'm wondering if we need to do any version adaptive or conditional xaml here.

Reply:
The comment in the XAML  below addresses this issue, partially. The AnimatedVisualPlayer relies on AnimationController APIs which are present only on RS4+. However, the AVP has a FallbackContent property that will instantiate its DataTemplate whenever its Source fails. In this example, the LottieVisualSource being used has a dependency on RS5+. To summarize: 

1.  when downlevel is invoked is up to the Source author and which version the IAnimatedVisualSource is able to target
2. the player always uses FallbackContent when it can't use its Source

I didn't want to have to outline this in the code sample. For now, the only Source type being ingested by the Player is provided by Lottie-Windows. As different Sources are create over time, they'll need to be documented accordingly.

Reply:
Gotcha, thanks for the explanation. To be a bit more specific - we're actively updating the gallery to run down to RS2. How will this sample behave if running on an RS2 machine? Will the app build if the minimum app target is RS2?

Reply:
It will render a static image instead of an animation on RS2/3/4

Reply:
Cool, thanks for the clarification.